test/test_combine_mul_sum.cpp

```diff
@@ -187,4 +187,44 @@ TEST_F(CombineMulSumAsMmaTest, AmpereMulSumToMatmul_Schedule) {
   }
 }
 
+TEST_F(CombineMulSumAsMmaTest, UseMatmulScheduler) {
+  // Keep multiples of 8 to keep vectorizable.
+  int M = 504, N = 136, K = 248;
+  const auto layout = MmaLayout::TT;
```

Comment:
Is there any value in looping over `kAllSupportedMmaLayout` in order to test the pattern matching in presence of various patterns?